.github/workflows/Release.yml

```diff
@@ -0,0 +1,61 @@
+name: Run release
+
+on:
+    push:
+        branches: [master]
+    pull_request:
+        branches: [master]
+
+jobs:
+    test:
+        name: "Release (${{ matrix.os }})"
+        runs-on: ${{ matrix.os }}
+        strategy:
+          matrix:
+            os: [ ubuntu-latest, windows-latest ]
+
+        steps:
+            - name: "Checking changed files"
+              id: files
+              uses: softprops/diffset@v1
+              if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository
+              with:
+                global_files: |
+                  !{packages/**/*,configs/e2e/*.json,scripts/**/*}
+              env:
+                GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            - name: "Checking-out code"
+              uses: actions/checkout@v2
+              with:
+                  submodules: false
+            - name: "Fetching master to compare"
+              if: github.event_name == 'pull_request' && steps.files.outputs.global_files == ''
+              run:
+                  git fetch --no-tags --prune --depth=1 origin +refs/heads/${{ github.base_ref }}:refs/remotes/origin/${{ github.base_ref }}
+            - name: "Defining environment variables"
+              if: startsWith(matrix.os, 'ubuntu')
+              id: variablesLinux
+              run:
+                  echo "::set-output name=arg::$(if [ "${{ steps.files.outputs.global_files }}" = "" ] && ${{ github.event_name == 'pull_request' }}; then echo '--since origin/${{ github.base_ref }}'; else echo ''; fi)"
+            - name: "Defining environment variables"
+              if: startsWith(matrix.os, 'windows')
+              id: variablesWindows
+              run:
+                echo "::set-output name=arg::$(If ('${{ steps.files.outputs.global_files }}' -eq '' -AND '${{ github.event_name == 'pull_request' }}' -eq 'true'){ echo '--since origin/${{ github.base_ref }}' } Else { echo '' })"
+            - name: "Defining cache"
+              uses: actions/cache@v2
+              env:
+                  cache-name: cache-node-modules
+              with:
+                  path: ~/.npm
+                  key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+            - name: "Defining node version"
+              uses: actions/setup-node@v2
+              with:
+                  node-version-file: '.nvmrc'
+            - name: "Installing dependencies"
+              run: npm install
+            - name: "Running release for production"
+              run: npx cross-env DEBUG="*" npm run release -- ${{ steps.variablesLinux.outputs.arg }} ${{ steps.variablesWindows.outputs.arg }}
```

Comment:
Is this `DEBUG` statement leftover or actually necessary? If latter, what does it do?

Reply:
Removed.